### PR TITLE
Added support for editorial stream

### PR DESCRIPTION
**Pull request by JayDaley**
@kesara please review

### CODE REVIEW

File: rfc7991bis.rnc [hunk 1]
```diff
@@ -28,7 +28,7 @@ rfc =
    attribute iprExtract { xsd:IDREF }?,
    [ a:defaultValue = "IETF" ]
    attribute submissionType {
-     "IETF" | "IAB" | "IRTF" | "independent"
+     "IETF" | "IAB" | "IRTF" | "independent" | "editorial"
    }?,
```
Comment (kesara) [inline rfc7991bis.rnc:32]: The `stream` element in line l1077 needs to have `editorial` as an option.

Comment (kesara) [inline rfc7991bis.rnc:32]: The `stream` attribute in `seriesInfo` in line 1099 needs to have `editorial` as an option too.

Comment (kesara) [reply rfc7991bis.rnc:32]: See `v3.rnc` changes in https://github.com/ietf-tools/xml2rfc/pull/958/files#diff-877886f739c9ef574984f93a4885203a458d76712d3505fe5011afaf711d8126

Comment (JayDaley) [reply rfc7991bis.rnc:32]: Now added.

Comment (JayDaley) [reply rfc7991bis.rnc:32]: Added

**Comment by JayDaley** [comment]
@kesara Please check again and merge if it's all correct now.

**Review by kesara — APPROVED** [review]
LGTM

**Pull request merged**